DFC.ServiceTaxonomy.ContentApproval/Drivers/ContentApprovalPartDisplayDriver.cs

```diff
@@ -245,62 +249,130 @@ private static bool IsCommentRequired(IUpdateModel updateModel)
              *  Cancel                      - N/A (exits page)
              */
 
-            if (updateModel != null && updateModel.ModelState != null && updateModel.ModelState?.Keys != null)
+            if (formSubmitAction != null)
+            {
+                var button = formSubmitAction.Item1;
```

Comment:
Can use (string button, string action) = formSubmitAction; instead of these 2 lines.  This is the new destructuring syntax.

Reply:
Changed to use (string button, string action)

Reply:
Didn't change here? just the other one.